pages/common/youtube-viewer.md

```diff
@@ -0,0 +1,20 @@
+# youtube-viewer
+
+> Command-line application for searching and playing videos from YouTube.
+> More information: <https://github.com/trizen/youtube-viewer>.
+
+- Search for a video:
+
+`youtube-viewer {{search_term}}`
+
+- Login to your YouTube account:
+
+`youtube-viewer --login`
+
+- Watch a video with a specific URL in VLC:
+
+`youtube-viewer --player={{vlc}} {{https://youtube.com/watch?v=1dastnond}}`
+
+- Get to a search prompt and play the selected video in 720p:
```

Comment:
```suggestion
- Get a search prompt and play the selected video in 720p:
```
Does this sound more nature?

Reply:
or `Open a search prompt`?

Reply:
`get` or `open` is okay with me.

Reply:
I agree with @zdroid - `Open` sounds better.
```suggestion
- Open a search prompt and play the selected video in 720p:
```

Reply:
It is a cli search.

Reply:
Changed to display.